### DISCUSSION TITLE
Pioneering InnerSource Case Studies for the Financial Service Industry

### DISCUSSION THREAD

**Discussion by mcleo-d** [finos/InnerSource#24] [Ideas]
As noted by @toshaellison - Within the InnerSource Special Interest Group is there appetite for doing one or more case studies focused on financial services experiences? 

@daniela-g-zheleva-db - I believe there could be a good opportunity within Deutsche Bank to partake in such a case study and would like to know if the InnerSource SIG would like to lead the initiative (@finos/innersource-maintainers) ..? 

The first case study could be something along the lines of "**Highlighting InnerSource success stories in financial services.**" which could include examples from Deutsche Bank, Capital One, RBC and JPMC for example.  

In terms of some of the goals for the SIG, this is a valuable tool in advocating for InnerSource, especially if someone could add cost saving figures. This could feed into the InnerSource SIG OSSF panel discussion too - https://github.com/finos/InnerSource/discussions/21

It would be great to get the thoughts of the SIG.

James.

**Comment by claredillon** [comment]
This was discussed in the InnerSource SIG meeting on 16th Aug 2021. It was suggested we might like to look at 2 different types of case studies:

1. Short reference-style case studies which can be used as evidence of InnerSource adoption
2. Longer case studies which look at thorny issues, and may remain anonymous.
